src/main/charts/bamboo-agent/Changelog.md

```diff
@@ -1,5 +1,21 @@
 # Change Log
 
+## 1.2.0
+
+**Release date:** 2022-02-10
```

Comment:
```suggestion
**Release date:** 2022-02-11
```

Can we update this for all products (if we are releasing today).

Reply:
If the release go to Monday let's update the release date then before the release.

Reply:
Updated~